### PR TITLE
Remove php in front of vendor/bin/php-cs-fixer fix, since that is wrong

### PR DESCRIPTION
**Pull request by madsoliver**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Just a minor change in the `README.md` that had one error about `php-cs-fixer` being run with `php` in front, that is wrong.
| Type?             | bug fix 
| BC breaks?        |  no
| Deprecations?     |  no
| Fixed ticket?     | 
| How to test?      | Agree/disagree that that feature is now correctly documented
| Possible impacts? | none

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,8 +51,8 @@ Running the tools can be done by calling its binary:
 
 ### PHP CS Fixer
 
-```php
-php vendor/bin/php-cs-fixer fix
+```bash
+$ vendor/bin/php-cs-fixer fix
```
Comment (matks) [inline README.md:55]: Actually it works 😄 

I think for most environments `php vendor/bin/php-cs-fixer` and `vendor/bin/php-cs-fixer` both work. But in some env. where the php binary is not registered globally or bash is not able to interpret that this is a PHP script it is needed to put `php` in front

Comment (madsoliver) [reply README.md:55]: Hi @matks 🙂

On my machine (Windows 11 WSL2 with Ubuntu and Docker with the official presta dockerfile) when I run `php vendor/bin/php-cs-fixer fix` I get the contents of the binary file in the terminal.

> or bash is not able to interpret that this is a PHP script

Is it a PHP-script though?
The source code of `vendor/bin/php-cs-fixer` is:

```
#!/usr/bin/env sh

dir=$(cd "${0%[/\\]*}" > /dev/null; cd '../friendsofphp/php-cs-fixer' && pwd)

if [ -d /proc/cygdrive ]; then
    case $(which php) in
        $(readlink -n /proc/cygdrive)/*)
            # We are in Cygwin using Windows php, so the path must be translated
            dir=$(cygpath -m "$dir");
            ;;
    esac
fi

"${dir}/php-cs-fixer" "$@"
```

**Review by kpodemski — APPROVED** [review]
I think we can accept it, but I agree with @matks that it should work properly in most scenarios.

Windows is weird ;-)

**Review by atomiix — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
I trust @kpodemski and @atomiix opinion: let's merge !

**Pull request merged**